competition/src/main/java/frc/robot/util/stateMachine/StateMachine.java

```diff
@@ -8,7 +8,7 @@
 
   private HashMap<String, Method> initStates;
   private HashMap<String, Method> runStates;
-  private String state = "";
+  protected String state = "";
```

Comment:
Please don't :(

Reply:
Why not? it's helpful to be able to know what the current state is

Reply:
I'd prefer adding a function to get the state, I don't want this being modified directly.